### PR TITLE
Add a parallel test case

### PR DESCRIPTION
**Pull request by smarterclayton**
[merge]

### CODE REVIEW

**Comment by liggitt** [comment]
weird test name in failure:
```
--- FAIL: TestBootstrapPolicyOverwritePolicyCommand-2 (8.12s)
	bootstrap_policy_test.go:95: unexpected error: <nil>
FAIL
```

**Comment by smarterclayton** [comment]
Hrm. Very hrm.  That's listing objects.  We should have been forbidden.  Race?

**Comment by liggitt** [comment]
see https://github.com/openshift/origin/issues/2956

**Comment by smarterclayton** [comment]
Going to deflake that sucker

**Comment by liggitt** [comment]
we're allowed until the policy delete propagates to the cache, because we were the cluster admin until that happened

**Comment by smarterclayton** [comment]
Ex-flaked

File: test/integration/bootstrap_policy_test.go [hunk 1]
```diff
@@ -91,9 +93,17 @@ func TestBootstrapPolicyOverwritePolicyCommand(t *testing.T) {
 		t.Errorf("unexpected error: %v", err)
 	}
 
-	if _, err := client.ClusterPolicies().List(labels.Everything(), fields.Everything()); err == nil || !kapierror.IsForbidden(err) {
-		t.Errorf("unexpected error: %v", err)
-	}
+	// after the policy is deleted, we must wait for it to be cleared from the policy cache
+	wait.Poll(10*time.Millisecond, 10*time.Second, func() (bool, error) {
```
Comment (liggitt) [inline test/integration/bootstrap_policy_test.go]: check for timeout err and fail the test

Comment (smarterclayton) [inline test/integration/bootstrap_policy_test.go]: too late

Comment (smarterclayton) [inline test/integration/bootstrap_policy_test.go]: This is like interactive pair programming

Comment (liggitt) [inline test/integration/bootstrap_policy_test.go]: just think of all the emails github has to send

Comment (liggitt) [inline test/integration/bootstrap_policy_test.go]: every

Comment (liggitt) [inline test/integration/bootstrap_policy_test.go]: single

Comment (liggitt) [inline test/integration/bootstrap_policy_test.go]: comment

Comment (smarterclayton) [inline test/integration/bootstrap_policy_test.go]: It's even more impressive to think they used to do it with Resque and Ruby.

**Pull request merged**